File: C#/kmeans/kmeans/Point.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace kmeans
{
    class Point
    {
        private float[] data;
        public Point(float[] data)
        {
            this.data = data;
        }
        public float[] getData()
        {
            return this.data;
        }
    }
}

File: cpp/kmeans/kmeans/Point.cpp

using namespace std;
#include<vector>

class Point {
	vector<float> data;
	
public:
	Point() {

	}
	Point(vector<float> data) {
		this->data = data;
	}
	Point(Point* point) {
		int a = 5;
	}
	vector<float> GetData() {
		return this->data;
	}
};